ui/lib/kv/addon/components/page/secret/metadata/version-history.hbs

```diff
@@ -77,15 +94,18 @@
                 @hasChevron={{false}}
                 data-test-popup-menu-trigger
               />
+              {{! TODO: replace }}
```

Comment:
Is this TODO going to be tackled in a follow-on PR? If so, could the comment be more descriptive?

Reply:
Oh, that was my note to self to change the link below, which I did. Will remove